xdsl/interpreters/experimental/pdl.py

```diff
@@ -209,17 +221,10 @@ def match_and_rewrite(self, xdsl_op: Operation, rewriter: PatternRewriter) -> No
 @dataclass
 class PDLFunctions(InterpreterFunctions):
     """
-    Applies the PDL pattern to all ops in the input `ModuleOp`. The rewriter
-    jumps straight to the last operation in the pattern, which is expected to
-    be a rewrite op. It creates an `AnonymousRewriter`, which runs on all
-    operations in the ModuleOp. For each operation, it determines whether the
-    operation fits the specified pattern and, if so, assigns the xDSL values
-    to the corresponding PDL SSA values, and runs the rewrite operations one by
-    one. The implementations in this class are for the RHS of the rewrite.
+    The implementations in this class are for the RHS of the rewrite.
```

Comment:
Can you add a more detailed documentation here?
What does this do? Is it the interpreter for a `pdl.rewrite`?
Has the operation already been matched? Where are they stored?